README.md

```diff
@@ -1,25 +1,24 @@
-# Modernisation Platform Terraform Module Template 
+# Modernisation Platform Terraform Module Template
```

Comment:
Title needs amending and the standards badge image and link below please!

Reply:
Done!